src/views/Suite.vue

```diff
@@ -71,50 +51,77 @@
       }
     },
     data: () => ({
-      pagination: {
-        rowsPerPage: 10
-      },
-      headers: [
+      columns: [
         {
-          sortable: true,
-          text: 'Task',
-          value: 'name'
+          property: 'name',
+          title: 'Task',
+          direction: null,
+          filterable: true,
+          collapseIcon: true
         },
         {
-          sortable: true,
-          text: 'State',
-          value: 'state'
+          property: 'state',
+          title: 'State',
+          direction: null,
+          filterable: false
         },
         {
-          sortable: true,
-          text: 'Host',
-          value: 'host'
+          property: 'host',
+          title: 'Host',
+          direction: null,
+          filterable: true
         },
         {
-          sortable: false,
-          text: 'Job ID',
-          value: 'jobId'
+          property: 'jobId',
+          title: 'Job ID',
+          direction: null,
+          filterable: true
         },
         {
-          sortable: false,
-          text: 'Latest Message',
-          value: 'latestMessage'
+          property: 'latestMessage',
+          title: 'Latest Message',
+          direction: null,
+          filterable: false
         },
         {
-          sortable: false,
-          text: 'Depth',
-          value: 'depth'
+          property: 'depth',
+          title: 'Depth',
+          direction: null,
+          filterable: false
         }
-      ]
+      ],
+      classes: {
+        table: "v-table",
+        '0/all': {'column text-xs-left': true}
+      },
+      filter: '',
+      start: 0,
+      end: 100,
+      polling: null
     }),
+    methods: {
+      filterChanged (filter) {
+        this.filter = filter;
+      },
+      fetchSuite() {
+        const suiteId = this.$route.params.name
+        suiteService.fetchSuiteTree(suiteId)
+        this.polling = setInterval(() => {
+          suiteService.currentTaskIndex += 1
+          suiteService.fetchSuiteTree(suiteId)
+        }, 3000)
```

Comment:
Comment on `3000ms` will be replaced by WebSocket.

Reply:
+1